usvg/src/main.rs

```diff
@@ -158,6 +160,8 @@ fn collect_args() -> Result<Args, pico_args::Error> {
         font_dirs:          input.values_from_str("--use-fonts-dir")?,
         skip_system_fonts:  input.contains("--skip-system-fonts"),
         list_fonts:         input.contains("--list-fonts"),
+        default_width:      input.opt_value_from_fn("--default-width", parse_length)?.unwrap_or(100),
```

Comment:
Add to HELP.

Reply:
https://github.com/RazrFalcon/resvg/blob/2fba92c4c8ea1d463b25c7ab82c225dc13bb5a19/usvg/src/main.rs#L13